computation-slurm-test/src/test/java/com/powsybl/computation/slurm/SlurmUnitTests.java

```diff
@@ -8,633 +8,44 @@
 
 import com.powsybl.commons.config.ModuleConfig;
 import com.powsybl.commons.config.YamlModuleConfigRepository;
-import com.powsybl.computation.*;
-import org.apache.commons.compress.archivers.zip.ZipArchiveEntry;
-import org.apache.commons.compress.archivers.zip.ZipArchiveOutputStream;
-import org.apache.commons.io.IOUtils;
-import org.junit.Assert;
+import com.powsybl.computation.ExecutionEnvironment;
 import org.junit.Before;
-import org.junit.Ignore;
-import org.junit.Test;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-import java.io.IOException;
-import java.io.InputStream;
-import java.io.OutputStream;
-import java.nio.file.Files;
-import java.nio.file.Path;
 import java.nio.file.Paths;
-import java.util.Arrays;
 import java.util.Collections;
-import java.util.List;
-import java.util.Objects;
-import java.util.concurrent.CompletableFuture;
-import java.util.concurrent.CompletionException;
-import java.util.concurrent.TimeUnit;
-import java.util.function.Supplier;
-import java.util.stream.IntStream;
-import java.util.zip.GZIPOutputStream;
-
-import static org.junit.Assert.assertFalse;
-import static org.junit.Assert.assertTrue;
 
 /**
  * @author Yichen TANG <yichen.tang at rte-france.com>
  */
-@Ignore
 public class SlurmUnitTests {
```

Comment:
Maybe it's an opportunity to rename this to `SlurmIntegrationTests`, I think the word "unit" is not appropriate because thoses tests acutally require a specific infrastructure !